markdown/ranger/ranger-integration-config.html.md.erb

```diff
@@ -124,8 +116,39 @@ Once the connection between HAWQ and Ranger is configured, you can either set up
 4. Click **Add Property...** and add the new property, `hawq_acl_type=ranger` property. (If the property already exists, change its value from `standalone` (the default) to `ranger`.)
 5. Click **Save** to save your changes.
 6. Select **Service Actions > Restart All** and confirm that you want to restart the HAWQ cluster.
-
 
-## <a id="caching"></a>Changing the Frequency of Policy Caching
+
+## <a id="customconfig"></a> Custom Configuration
+
+Configuration files for the HAWQ Ranger Plug-in Service are located in the `$GPHOME/ranger/etc` directory. These files include:
+
+| File     |  Description     |
+|-------------|---------------------------|
+| ranger-hawq-audit.xml |  HAWQ Ranger audit-related configuration, including the audit provider (log4j, Solr, HDFS) and provider-specific configuration |
+| ranger-hawq-security.xml |  HAWQ Ranger service configuration, including the policy change polling interval |
+| rps.properties |  HAWQ Ranger deployment-related configuration, including the RPS port definition and JVM parameters|
+
+You should not need to modify RPS configuration in a default installation.  Any configuration changes you do make after you have registered the HAWQ Ranger Plug-in requires a restart of the service. You can either restart the HAWQ cluster or restart just the HAWQ Ranger Plug-in Service:
```

Comment:
Get rid of RPS acronym.  I'm also not sure that we want to say "you should not need to modify..."  Auditing, polling interval seem like pretty common changes that might be made.